### ISSUE TITLE
Crash when trying to play a video, after upgrading server to 10.7.5

### ISSUE THREAD

**Issue by Salamandar** [jellyfin/jellyfin-androidtv#885]
I don't know what I could add in the description.
The video plays correctly in Jellyfin-web. 

I also tried playing files that i *could* play before the server update (from 10.7.0) and the android-tv app still crashes.

**Comment by ZachMayry** [comment]
Getting something similar. I had to hard code my docker server tag to 10.7.2

**Comment by Harteex** [comment]
Same problem here after updating the server to 10.7.5. Before that I used 10.6.x and the same video played fine.

I've already submitted crash reports from the app, but here's also the server log when trying to play a video if that helps.

```
[2021-05-17 20:22:10.519 +02:00] [INF] [9] Jellyfin.Api.Controllers.MediaInfoController: GetPostedPlaybackInfo profile: DeviceProfile { Name: "Android-VLC", Id: null, Identification: null, FriendlyName: null, Manufacturer: null, ManufacturerUrl: null, ModelName: null, ModelDescription: null, ModelNumber: null, ModelUrl: null, SerialNumber: null, EnableAlbumArtInDidl: False, EnableSingleAlbumArtLimit: False, EnableSingleSubtitleLimit: False, SupportedMediaTypes: "Audio,Photo,Video", UserId: null, AlbumArtPn: null, MaxAlbumArtWidth: 0, MaxAlbumArtHeight: 0, MaxIconWidth: null, MaxIconHeight: null, MaxStreamingBitrate: 20000000, MaxStaticBitrate: 100000000, MusicStreamingTranscodingBitrate: 128000, MaxStaticMusicBitrate: null, SonyAggregationFlags: null, ProtocolInfo: null, TimelineOffsetSeconds: 0, RequiresPlainVideoItems: False, RequiresPlainFolders: False, EnableMSMediaReceiverRegistrar: False, IgnoreTranscodeByteRangeRequests: False, XmlRootAttributes: [], DirectPlayProfiles: [DirectPlayProfile { Container: "m4v,3gp,ts,mpegts,mov,xvid,vob,mkv,wmv,asf,ogm,ogv,m2v,mpg,mpeg,mp4,webm,wtv", AudioCodec: "aac,mp3,mp2,ac3,wma,wmav2,dca,dts,pcm,PCM_S16LE,PCM_S24LE,opus,flac,truehd", VideoCodec: null, Type: Video }, DirectPlayProfile { Container: "flac,aac,mp3,mpa,wav,wma,mp2,ogg,oga,webma,ape", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "jpg,jpeg,png,gif", AudioCodec: null, VideoCodec: null, Type: Photo }], TranscodingProfiles: [TranscodingProfile { Container: "mkv", Type: Video, VideoCodec: "h264", AudioCodec: "aac,mp3", Protocol: null, EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: True, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: null, MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "mp3", Type: Audio, VideoCodec: null, AudioCodec: "mp3", Protocol: null, EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: null, MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }], ContainerProfiles: [], CodecProfiles: [CodecProfile { Type: Video, Conditions: [ProfileCondition { Condition: NotEquals, Property: VideoProfile, Value: "none", IsRequired: False }], ApplyConditions: [], Codec: "hevc", Container: null }, CodecProfile { Type: Video, Conditions: [ProfileCondition { Condition: EqualsAny, Property: VideoProfile, Value: "high|main|baseline|constrained baseline", IsRequired: False }, ProfileCondition { Condition: LessThanEqual, Property: VideoLevel, Value: "51", IsRequired: False }, ProfileCondition { Condition: GreaterThanEqual, Property: RefFrames, Value: "2", IsRequired: False }], ApplyConditions: [], Codec: "h264", Container: null }, CodecProfile { Type: VideoAudio, Conditions: [ProfileCondition { Condition: LessThanEqual, Property: AudioChannels, Value: "8", IsRequired: False }], ApplyConditions: [], Codec: null, Container: null }], ResponseProfiles: [], SubtitleProfiles: [SubtitleProfile { Format: "srt", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "srt", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "subrip", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ass", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ssa", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "pgs", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "pgssub", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "dvdsub", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "vtt", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "sub", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "smi", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "idx", Method: Embed, DidlMode: null, Language: null, Container: null }] }
[2021-05-17 20:22:10.849 +02:00] [INF] [9] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "Harteex". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: False EnableAudioPlaybackTranscoding: True
[2021-05-17 20:22:10.855 +02:00] [INF] [9] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android-VLC", Path: "D:\<redacted>.avi", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-05-17 20:22:10.864 +02:00] [INF] [9] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android-VLC", No video direct play profiles found for "D:\<redacted>.avi" with codec "mpeg4"
[2021-05-17 20:22:10.874 +02:00] [INF] [9] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android-VLC", Path: "D:\<redacted>.avi", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-05-17 20:22:10.884 +02:00] [INF] [9] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android-VLC", No video direct play profiles found for "D:\<redacted>.avi" with codec "mpeg4"
[2021-05-17 20:22:11.090 +02:00] [INF] [18] Jellyfin.Api.Controllers.MediaInfoController: GetPostedPlaybackInfo profile: DeviceProfile { Name: "Android-Exo", Id: null, Identification: null, FriendlyName: null, Manufacturer: null, ManufacturerUrl: null, ModelName: null, ModelDescription: null, ModelNumber: null, ModelUrl: null, SerialNumber: null, EnableAlbumArtInDidl: False, EnableSingleAlbumArtLimit: False, EnableSingleSubtitleLimit: False, SupportedMediaTypes: "Audio,Photo,Video", UserId: null, AlbumArtPn: null, MaxAlbumArtWidth: 0, MaxAlbumArtHeight: 0, MaxIconWidth: null, MaxIconHeight: null, MaxStreamingBitrate: 20000000, MaxStaticBitrate: 100000000, MusicStreamingTranscodingBitrate: 128000, MaxStaticMusicBitrate: null, SonyAggregationFlags: null, ProtocolInfo: null, TimelineOffsetSeconds: 0, RequiresPlainVideoItems: False, RequiresPlainFolders: False, EnableMSMediaReceiverRegistrar: False, IgnoreTranscodeByteRangeRequests: False, XmlRootAttributes: [], DirectPlayProfiles: [DirectPlayProfile { Container: "m4v,mov,xvid,vob,mkv,wmv,asf,ogm,ogv,mp4,webm", AudioCodec: "aac,ac3,eac3,aac_latm,mp3,mp2,dca,dts", VideoCodec: "h264,hevc,vp8,vp9,mpeg,mpeg2video", Type: Video }, DirectPlayProfile { Container: "aac,mp3,mpa,wav,wma,mp2,ogg,oga,webma,ape,opus", AudioCodec: null, VideoCodec: null, Type: Audio }, DirectPlayProfile { Container: "jpg,jpeg,png,gif", AudioCodec: null, VideoCodec: null, Type: Photo }], TranscodingProfiles: [TranscodingProfile { Container: "mkv", Type: Video, VideoCodec: "h264", AudioCodec: "ac3,aac,mp3", Protocol: null, EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: True, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: null, MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }, TranscodingProfile { Container: "mp3", Type: Audio, VideoCodec: null, AudioCodec: "mp3", Protocol: null, EstimateContentLength: False, EnableMpegtsM2TsMode: False, TranscodeSeekInfo: Auto, CopyTimestamps: False, Context: Streaming, EnableSubtitlesInManifest: False, MaxAudioChannels: null, MinSegments: 0, SegmentLength: 0, BreakOnNonKeyFrames: False }], ContainerProfiles: [], CodecProfiles: [CodecProfile { Type: Video, Conditions: [ProfileCondition { Condition: EqualsAny, Property: VideoProfile, Value: "high|main|baseline|constrained baseline", IsRequired: False }, ProfileCondition { Condition: LessThanEqual, Property: VideoLevel, Value: "51", IsRequired: False }], ApplyConditions: [], Codec: "h264", Container: null }, CodecProfile { Type: Video, Conditions: [ProfileCondition { Condition: LessThanEqual, Property: RefFrames, Value: "12", IsRequired: False }], ApplyConditions: [ProfileCondition { Condition: GreaterThanEqual, Property: Width, Value: "1200", IsRequired: False }], Codec: "h264", Container: null }, CodecProfile { Type: Video, Conditions: [ProfileCondition { Condition: LessThanEqual, Property: RefFrames, Value: "4", IsRequired: False }], ApplyConditions: [ProfileCondition { Condition: GreaterThanEqual, Property: Width, Value: "1900", IsRequired: False }], Codec: "h264", Container: null }, CodecProfile { Type: Video, Conditions: [ProfileCondition { Condition: NotEquals, Property: VideoProfile, Value: "none", IsRequired: False }], ApplyConditions: [], Codec: "hevc", Container: null }, CodecProfile { Type: VideoAudio, Conditions: [ProfileCondition { Condition: LessThanEqual, Property: AudioChannels, Value: "6", IsRequired: False }], ApplyConditions: [], Codec: null, Container: null }], ResponseProfiles: [], SubtitleProfiles: [SubtitleProfile { Format: "srt", Method: External, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "srt", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "subrip", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ass", Method: Encode, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "ssa", Method: Encode, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "pgs", Method: Encode, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "pgssub", Method: Encode, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "dvdsub", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "vtt", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "sub", Method: Embed, DidlMode: null, Language: null, Container: null }, SubtitleProfile { Format: "idx", Method: Embed, DidlMode: null, Language: null, Container: null }] }
[2021-05-17 20:22:11.395 +02:00] [INF] [18] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "Harteex". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: False EnableAudioPlaybackTranscoding: True
[2021-05-17 20:22:11.399 +02:00] [INF] [18] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android-Exo", Path: "D:\<redacted>.avi", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-05-17 20:22:11.403 +02:00] [INF] [18] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android-Exo", No video direct play profiles found for "D:\<redacted>.avi" with codec "mpeg4"
[2021-05-17 20:22:11.405 +02:00] [INF] [18] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android-Exo", Path: "D:\<redacted>.avi", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-05-17 20:22:11.409 +02:00] [INF] [18] Jellyfin.Api.Helpers.MediaInfoHelper: Profile: "Android-Exo", No video direct play profiles found for "D:\<redacted>.avi" with codec "mpeg4"
[2021-05-17 20:22:11.702 +02:00] [WRN] [23] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS "192.168.1.1" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."
[2021-05-17 20:22:11.738 +02:00] [INF] [23] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.1.1" closed
```

**Comment by Sundown89** [comment]
I am also experiencing this. In my case it only seems to occur when playing a file through my reverse proxy (external). Same file plays fine on the same device (Nvidia Shield) from my internal network. 

```
[2021-05-17 12:34:59.895 +02:00] [INF] User policy for "Lars". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2021-05-17 12:34:59.896 +02:00] [INF] Profile: "Android-Exo", Path: "<nfs://path/to/file.mkv>", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-05-17 12:34:59.896 +02:00] [INF] Profile: "Android-Exo", Path: "<nfs://path/to/file.mkv>", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-05-17 12:34:59.897 +02:00] [INF] Profile: "Android-Exo", Path: "<nfs://path/to/file.mkv>", isEligibleForDirectPlay: True, isEligibleForDirectStream: True
[2021-05-17 12:35:00.258 +02:00] [INF] Playback Started
[2021-05-17 12:35:12.980 +02:00] [INF] Sending ForceKeepAlive message to 1 inactive WebSockets.
[2021-05-17 12:35:36.980 +02:00] [INF] Sending ForceKeepAlive message to 1 inactive WebSockets.
[2021-05-17 12:35:48.982 +02:00] [INF] Sending ForceKeepAlive message to 1 inactive WebSockets.
[2021-05-17 12:36:00.981 +02:00] [INF] Lost 1 WebSockets.
[2021-05-17 12:42:08.756 +02:00] [WRN] WS "<external.ip.1.1>" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."
[2021-05-17 12:42:08.778 +02:00] [INF] WS "<external.ip.1.1>" closed
[2021-05-17 12:42:12.132 +02:00] [INF] WS "<external.ip.1.1>" request
```

**Comment by Aninstance** [comment]
Same issue. 
`[2021-05-19 14:54:37.949 +00:00] [WRN] [82] Emby.Server.Implementations.HttpServer.WebSocketConnection: WS "192.168.1.9" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."`

**Comment by Salamandar** [comment]
I tried to fully reinstall the server and it solved the issue. I don't think this is an acceptable workaround though.

**Comment by Aninstance** [comment]
> I tried to fully reinstall the server and it solved the issue. I don't think this is an acceptable workaround though.

Thanks for the advice @Salamandar, I'll try that myself & report back.

**Comment by nielsvanvelzen** [comment]
Closing as duplicate of jellyfin/jellyfin#6034 (fixed in next server release)

**Comment by Harteex** [comment]
It seems like it's fixed for the next 10.7 server release, but the change will remain for 10.8 and will have to be fixed in the clients by then.
https://github.com/jellyfin/jellyfin/pull/6022
